ExampleApp Android/build.gradle.kts

```diff
@@ -50,4 +55,5 @@ dependencies {
     //test libs
     testImplementation(Dependencies.jvmTestLibraries)
     androidTestImplementation(Dependencies.androidTestLibraries)
+    implementation("com.google.guava:guava:31.1-jre")
```

Comment:
It would be good to move this dependency to the Dependencies class, what do you think?